src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -980,6 +997,16 @@ const FormCreationForm = ({formUuid, formUrl, formHistoryUrl}) => {
     });
   }, []);
 
+  useAsync(async () => {
+    const reusableFormDefinitions = await loadFromBackend([
+      {endpoint: FORM_DEFINITIONS_ENDPOINT, query: {is_reusable: true}},
+    ]);
+    dispatch({
+      type: 'REUSABLE_FORM_DEFINITIONS_LOADED',
+      payload: reusableFormDefinitions[0],
```

Comment:
what is this magic `[0]` index doing?

Reply:
`loadFromBackend` returns an array containing the results of each request. In our case, we are only making one request

Reply:
right - my first hunch was to just use `get` directly, but that doesn't handle pagination - in this case I'd destructure the return value to increase readability:

```js
const response = await loadFromBackend(...);
const [reusableFormDefinitions] = responses;
```